src/coreclr/tools/aot/Mono.Linker.Tests.Cases/Mono.Linker.Tests.Cases.csproj

```diff
@@ -8,6 +8,8 @@
     <DefineConstants>$(DefineConstants);INCLUDE_EXPECTATIONS</DefineConstants>
     <WarningLevel>0</WarningLevel>
     <AnalysisLevel>0</AnalysisLevel>
+    <RunAnalyzers>true</RunAnalyzers>
+    <DefineConstants>$(DefineConstants);NATIVEAOT</DefineConstants>
```

Comment:
I know this is handy when writing the code, but do we need this to build the tests?
My point is that once the source trees are shared, there should be no reason to build the test cases assembly twice, since it actually isn't used as-is. We rebuild each test separately at runtime - and that build already defines `NATIVEAOT`.